yarn.lock

```diff
@@ -19587,7 +19587,7 @@ selenium-webdriver@^3.6.0:
   resolved "https://registry.yarnpkg.com/selenium-webdriver/-/selenium-webdriver-3.6.0.tgz#2ba87a1662c020b8988c981ae62cb2a01298eafc"
   integrity sha512-WH7Aldse+2P5bbFBO4Gle/nuQOdVwpHMTL6raL3uuBj/vPG07k6uzt3aiahu352ONBr5xXh0hDlM3LhtXPOC4Q==
   dependencies:
-    jszip "^3.1.3"
+    jszip "^3.5.0"
```

Comment:
This line shouldn't be changed

Reply:
I don't know what you mean here. selenium-webdriver 3.6.0 is using jszip 3.5.0.  and there is no other application using jszip 3.1.3.

Reply:
I made a little change. You should check again